Add sv_useGameLiftServer info to AWSGameLift local testing page (#2309)
o3de/o3de.org: content/docs/user-guide/gems/reference/aws/aws-gamelift/local-testing.md

```

## 2. Set `sv_useGameLiftServer` to `true`

> To prevent the server from trying to reach out to Amazon GameLift while running locally, the AWS GameLift gem server manager will not initialize unless you opt-in to using it by setting the `sv_useGameLiftServer` CVAR to `true`. You can set this value by creating a settings registry file named **commands.MYPROJECT_serverlauncher.setreg** in the **YourProject/Registry** folder with the following contents:

Author: >If we have other cvars/registry settings for this gem, they should be called out in settings page.

The only other CVAR I found was `cl_gameliftLocalEndpoint`, which is used in Step 4 on this page. It doesn't look like there's a "settings" page for this gem, or for any other AWS gem, is this somewhere else in the docs?

> To prevent the server from trying to reach out to Amazon GameLift while running locally, the AWS GameLift gem server manager will not initialize unless you opt-in to using it by setting the `sv_useGameLiftServer` CVAR to `true`. You can set this value by creating a settings registry file named **commands.MYPROJECT_serverlauncher.setreg** in the **YourProject/Registry** folder with the following contents:

Reviewer: @allisaurus There are no other instances of `cl_gamelift*` CVARs in the docs. Any CVARs that users can use in O3DE I agree should be documented

```json
{
    "Amazon":
    {
        "AzCore":
        {
            "Runtime":
            {
                "ConsoleCommands":
                {
                    "sv_useGameLiftServer": "true"
                } 
            } 
        } 
    } 
}
```

> This will let your game server use GameLift local when running in the Editor with `ctrl` + `g` or when running the server launcher directly (see next step).

Reviewer 2: Nitpick: You might want to change this to "running in the Editor when entering game mode", I'm pretty sure control+g is a key binding that can be customized. If they're using the UI to enter/exit game mode by clicking the play arrow (in the top right, by default), this might not be as clear.

Also, how does this system work when entering the non-game simulation mode? That's the little bouncing ball that's to the top right (in the default UI layout) to the right of the enter/exit game mode arrow. It's a mode that initializes some game systems, but not all. Do servers get spun up, does multiplayer run in some fashion in simulation mode, or is most of that disabled? If it's all disabled, no need to mention it. If it is enabled, then it's probably worth including a note on that in the documentation here, too.

```

## 2. Set `sv_useGameLiftServer` to `true`

> When you are ready to test with GameLift, opt-in by setting the `sv_useGameLiftServer` CVAR to `true`. This CVAR is off by default to facilitate local testing prior to the integration of the GameLift Server SDK into your game server.

Reviewer 3: ```suggestion
When you are ready to test with GameLift, opt-in by setting the `sv_useGameLiftServer` CVAR to `true`. This CVAR is off by default to facilitate local testing, prior to the integration of the GameLift Server SDK into your game server.
```

Author: this actually reads more intuitively to me w/o the comma

> You can set this value by creating a settings registry file named **commands.MYPROJECT_serverlauncher.setreg** in the **YourProject/Registry** folder with the following contents:

Reviewer: - "YourProject" --> "MyProject", so it's clear we're talking about the same project. 
I also wrote them the both in camel case, please verify if this is correct. Should the project name in the `.setreg` file be all upper case?
- Use `code-style` formatting for files and paths

```suggestion
You can set this value by creating a settings registry file named `commands.MyProject_serverlauncher.setreg` in the `MyProject/Registry` folder with the following contents:
```

```json
{
    "Amazon":
    {
        "AzCore":
        {
            "Runtime":
            {
                "ConsoleCommands":
                {
                    "sv_useGameLiftServer": "true"
                } 
            } 
        } 
    } 
}
```

> This will let your game server use GameLift local when running game or simulation mode in the Editor, or when running the server launcher directly (see next step).

Reviewer: ```suggestion
This lets your game server use GameLift local when running game or simulation mode in the Editor, or when running the server launcher directly (see next step).
```
